tests/integration/integration.go

```diff
@@ -622,7 +618,20 @@ var _ = ginkgo.Describe("[Tx Processing]", ginkgo.Serial, func() {
 	})
 })
 
-func expectBlk(i instance) func(add bool) []*chain.Result {
+var _ = ginkgo.Describe("[Custom VM Tests]", func() {
+	require := require.New(ginkgo.GinkgoT())
+
+	for testRegistry := range registry.GetTestsRegistries() {
```

Comment:
Could we pass the test registry into the integration/e2e tests rather than using a global registry this way?

Otherwise, I think this is just adding an extra step to the same pattern as https://github.com/ava-labs/hypersdk/pull/1667#discussion_r1815932572

Reply:
Unfortunately, that won't work with ginkgo. we need the tests to be available in the `GetTestRegistries` ( or another global variable ) by the time this method is being invoked.
What I can do is to set this to be a function placeholder that would later on invoke the various tests. However, that would not be a `ginkgo` node and any failure on these tests would not be reported correctly.